test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AttributeRouteTest.cs

```diff
@@ -545,6 +545,118 @@ public void AttributeRoute_ThrowsRouteCreationException_ForConstraintsNotTakingA
             Assert.IsType<RouteCreationException>(exception.InnerException);
         }
 
+        [Fact]
+        public void GetEntries_DoesNotCreateOutboundEntriesForAttributesWithSuppressForLinkGenerationSetToTrue()
```

Comment:
Would like to see a case for outbound with two actions with the same route values - and one of them is suppressed.